### PR TITLE
DMP-1271 - added rest endpoint to get count of non accessed media req…

### PR DESCRIPTION
**Pull request by HMCTSDeveloper**
https://tools.hmcts.net/jira/browse/DMP-1328

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestNonAccessedCountIntTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+
+import java.net.URI;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
```
Comment (tonymort) [inline src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestNonAccessedCountIntTest.java:20]: This should inherited from IntegrationBase

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestNonAccessedCountIntTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+@ActiveProfiles({"intTest", "h2db"})
+@AutoConfigureMockMvc
+@Slf4j
+@Transactional
```
Comment (tonymort) [inline src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestNonAccessedCountIntTest.java:24]: We need to move away from Transactional integration tests.

File: src/main/java/uk/gov/hmcts/darts/audio/service/MediaRequestService.java [hunk 3]
```diff
@@ -12,6 +15,9 @@
 
 public interface MediaRequestService {
 
+    @Transactional(propagation = Propagation.SUPPORTS)
```
[the remark this answers is not in the document]
Comment (HMCTSDeveloper) [reply src/main/java/uk/gov/hmcts/darts/audio/service/MediaRequestService.java:18]: ACCEPT

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestNonAccessedCountIntTest.java (hunk 2 above)
Comment (HMCTSDeveloper) [reply src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestNonAccessedCountIntTest.java:24]: ACCEPT

**Review by tonymort — APPROVED** [review]

**Pull request merged**